File: c/code/118_pattern_10.c
#include<stdio.h>
int main() {
    int n;
    printf("Enter n :- ");
    scanf("%d",&n);
    for(int i=0;i<n;i++) {
        for(int j=n-1;j>i;j--) {
            printf(" ");
        }
        printf("/");
        for(int k=0;k<2*i;k++) {
            if(k%2 == 0) printf("/");
            else printf("\\");
        }
        printf("\\");
        printf("\n");
    }
}

/*

Output :-
Enter n :- 5
    /\
   //\\
  //\/\\
 //\/\/\\
//\/\/\/\\

*/

File: python/code/118_pattern10.py
n = int(input("Enter n :- "))
for i in range(n):
    for j in range(n-1,i,-1):
        print(" ", end='')
    print("/", end='')
    for k in range(2*i):
        if(k%2 == 0):
            print("/", end='');
        else:
            print("\\", end='');        
    print("\\")
    
# Output :-
# Enter n :- 5
#     /\
#    //\\
#   //\/\\
#  //\/\/\\
# //\/\/\/\\  